python-template/{{cookiecutter.placeholder_repo_name}}/.editorconfig

```diff
@@ -2,7 +2,7 @@
 # ** DO NOT EDIT THIS FILE **
```

Comment:
Another one! :dizzy_face:

Reply:
oh noooo the curse of `.editorconfig` 😆

Reply:
thankfully just the one! easy and fast, pushed change.